cmd/kpod/spec.go

```diff
@@ -17,6 +17,27 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func addPidNS(config *createConfig, g *generate.Generator) error {
```

Comment:
Can you add a TODO here to switch to the libpod support for sharing namespaces when that lands?